examples/BuddyLlama/CMakeLists.txt

```diff
@@ -63,14 +63,4 @@ SET_TARGET_PROPERTIES(
 
 add_executable(buddy-llama-run llama-main.cpp)
 target_link_directories(buddy-llama-run PRIVATE ${LLVM_MLIR_LIBRARY_DIR})
-
-set(BUDDY_LLAMA_LIBS
-  LLAMA
-  mlir_c_runner_utils
-  omp
-)
-if(BUDDY_MLIR_USE_MIMALLOC)
-  list(APPEND BUDDY_LLAMA_LIBS mimalloc)
-endif()
-
-target_link_libraries(buddy-llama-run ${BUDDY_LLAMA_LIBS})
+target_link_libraries(buddy-llama-run LLAMA)
```

Comment:
The `LLAMA` target linked libraries retain the previous version. (use `BUDDY_LLAMA_LIBS`)

Reply:
OK